src/Hl7.Fhir.Core/Model/Generated/Template-ModelInfo.cs

```diff
@@ -962,7 +962,7 @@ public static string Version
-                new SearchParamDefinition() { Resource = "DiagnosticReport", Name = "encounter", Description = @"The Encounter when the order was made", Type = SearchParamType.Reference, Path = new string[] { "DiagnosticReport.context", }, Target = new ResourceType[] { ResourceType.Encounter, ResourceType.EpisodeOfCare, }, XPath = "f:DocumentReference/f:context/f:encounter | f:RiskAssessment/f:context | f:DeviceRequest/f:context | f:Procedure/f:context | f:List/f:encounter | f:VisionPrescription/f:encounter | f:ProcedureRequest/f:context | f:Flag/f:encounter | f:Observation/f:context | f:DiagnosticReport/f:context | f:NutritionOrder/f:encounter | f:Composition/f:encounter", Expression = "DocumentReference.context.encounter | RiskAssessment.context | DeviceRequest.context | Procedure.context | List.encounter | VisionPrescription.encounter | ProcedureRequest.context | Flag.encounter | Observation.context | DiagnosticReport.context | NutritionOrder.encounter | Composition.encounter", Url = "http://hl7.org/fhir/SearchParameter/clinical-encounter" },
+                new SearchParamDefinition() { Resource = "DiagnosticReport", Name = "encounter", Description = @"The Encounter when the order was made", Type = SearchParamType.Reference, Path = new string[] { "DiagnosticReport.encounter", }, Target = new ResourceType[] { ResourceType.Encounter, }, XPath = "f:DocumentReference/f:context/f:encounter | f:RiskAssessment/f:context | f:DeviceRequest/f:context | f:Procedure/f:context | f:List/f:encounter | f:VisionPrescription/f:encounter | f:ProcedureRequest/f:context | f:Flag/f:encounter | f:Observation/f:context | f:DiagnosticReport/f:context | f:DiagnosticReport/f:encounter | f:NutritionOrder/f:encounter | f:Composition/f:encounter", Expression = "DocumentReference.context.encounter | RiskAssessment.context | DeviceRequest.context | Procedure.context | List.encounter | VisionPrescription.encounter | ProcedureRequest.context | Flag.encounter | Observation.context | DiagnosticReport.context | DiagnosticReport.encounter | NutritionOrder.encounter | Composition.encounter", Url = "http://hl7.org/fhir/SearchParameter/clinical-encounter" },
```

Comment:
Should the Path not be `Path = new string[] { "DiagnosticReport.context" }` ?
Only the target should be `ResourceType.Encounter`, right?